### PR TITLE
20181129 1106

### PR DESCRIPTION
**Pull request by farhanpatel**

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -2717,6 +2540,61 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Client/Intro.strings" source-language="en" datatype="plaintext">
```
Comment (farhanpatel) [inline templates/firefox-ios.xliff]: These have moved but the filename and the ids should be the same.

Comment (Delphine) [reply templates/firefox-ios.xliff]: Hmm Github is being super weird to me today.
So once more: yes, confirming this is good!

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -620,13 +465,21 @@
...
-        <source>Recommended by Pocket</source>
+        <source>Trending on Pocket</source>
         <note>Section title label for Recommended by Pocket section</note>
       </trans-unit>
       <trans-unit id="ActivityStream.Pocket.Trending">
         <source>Trending</source>
         <note>The description of a Pocket Story</note>
       </trans-unit>
+      <trans-unit id="ActivityStream.TopSites.RowCount">
+        <source>%d rows</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff]: This fell through the cracks.
Unfortunately, I don't think we support plurals correctly on iOS (see https://bugzilla.mozilla.org/show_bug.cgi?id=1181777). So we will have to find a workaround, like "Rows: %d" instead (we've had to do this in the past)

**Comment by farhanpatel** [comment]
fixed

**Pull request closed without merging**